readme/template-guide-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template Guide
+
+Read this document before you start working on the README template.
+
+
+*Note: As your project evolves and grows, so should your README file. READMEs should be revisited regularly to ensure relevant information is captured for new users.*
+
+## Why do I need a README?
+READMes are the first document users find and read. All projects include a README, and users will expect to find one in your project's root folder. A README tells users what they need to know about your project, if the project is relevant to the user, and how users can engage with the project. Your README is your project's first impression.
+
+In some cases, larger projects may include sub-READMEs, or separate, smaller READMEs that support subfolders in the project.
+
+## Content of the README template guide
+The following sections provide guidance and context on how to fill out the README template.
+
+### About the "Project logo and badge" section
+This section is optional. 
+
+A project logo can be helpful for users to visually identify your project. 
+
+Embed relevant badges to your README to add credibility to your project. Badges highlight important information and can help users determine project fit quickly. [Learn more about badges.](https://github.com/badges/shields)
+
+### About the "Project name" section
+The project name should be prominent in the README and easy for users to identify. You may want to consider adding the following to help identify the project:
+- Project URL
+- Name(s) of project owner(s)
+
+*Note: Consider the stage your project is at and how much information is required in your README. Names associated with the project may be better located in a separate contact document.*
```

Comment:
**Suggestion (non-blocking):** If you want, we could mention that we have a template for the contact document and link to that. It's the new Our Team template that I have an open pull request for.

Reply:
I think it's great to link to other templates to encourage more use of our templates. I'll add it in once the template is available.

Reply:
Added!